common/netns.go

```diff
@@ -79,3 +79,23 @@ func NewNetNsContext(path string) (*NetNSContext, error) {
 		newns:  newns,
 	}, nil
 }
+
+// ExecInNs call the given function in the context of the given netns
+func ExecInNs(nsPath string, fnc func(origns, newns netns.NsHandle) error) error {
```

Comment:
It seems we don't actually jump into the namespace. So `ExecInNs` is a bit misleading

Reply:
indeed and not used anymore, going to revert this